install_config/topics/configuring_for_gce-proc.adoc

```diff
@@ -0,0 +1,159 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+= Configuring {product-title} for GCP using Ansible
+
+You can configure {product-title} for Google Compute Platform (GCP) at
+installation time or by running the
+xref:../install/configuring_inventory_file.adoc#configuring-ansible[Ansible
+inventory file] after installation. 
+
+You must define the `openshift_cloudprovider_kind`, `openshift_gcp_project` and
+`openshift_gcp_prefix` parameters. For
+xref:../configuring_gce.adoc#gce-load-balancer[running load balancer services
+using GCP], the Compute Engine VM node instances require the `ocp` suffix:
+`<openshift_gcp_prefix>ocp`. For example, if the value of the
+`openshift_gcp_prefix` parameter is set to `mycluster`, you must tag the nodes
+with `myclusterocp`. See
+link:https://cloud.google.com/vpc/docs/add-remove-network-tags[Adding and
+Removing Network Tags] for more information on how to add network tags to
+Compute Engine VM instances.
+
+Use the following when using the advanced installation to configure your
+{product-title} environment for GCP:
+
+[source,yaml]
+----
+[OSEv3:vars]
+# GCP Provider Configuration
+openshift_cloudprovider_kind=gce
+openshift_gcp_project=<projectid> <1>
+openshift_gcp_prefix=<uid> <2>
+# If deploying single zone cluster set to "False"
+openshift_gcp_multizone=False <3>
+openshift_gcp_network_name=<network name> <4>
+----
+<1> Set to the GCE project ID where the existing instances are running.
+<2> Set to a unique string to identify each {product-title} cluster.
+<3> Set to `true` to trigger xref:configuring-gce-multizone[multizone deployments] on GCE. Set to `False` by
+default.
+<4> Set to the network name if not using the default.
+
+Installing with Ansible also creates the following files for you:
```

Comment:
I'd say:

Installing with Ansible also creates and configures the following files properly to fit the GCE environment for you: